git-shame

```diff
@@ -2,8 +2,37 @@
 
 require "optparse"
 
+color_value = nil
+
+def get_setting(name)
+  IO.popen("git config --get " + name) { |pipe|
+    return pipe.read.strip
+  }
+end
+
+['color.shame', 'color.ui'].each { |name|
+  setting = get_setting name
+
+  puts name, setting
+  case setting
+  when 'alway'
```

Comment:
This ought to be "always".

Reply:
Oh, indeed, thanks for pointing out! It is fixed now.